src/getCoinCombination.test.js

```diff
@@ -9,12 +9,30 @@ describe(`Function 'getCoinCombination':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(getCoinCombination(1)).toBeInstanceOf(Array);
   });
 
   it(`should return [0, 0, 0, 0] if amount = 0`, () => {
+    expect(getCoinCombination(0)).toEqual([0, 0, 0, 0]);
+  });
 
+  it(`should return [2, 0, 0, 0] if amount = 2`, () => {
+    expect(getCoinCombination(2)).toEqual([2, 0, 0, 0]);
   });
 
-  // write more tests here
+  it(`should return [0, 0, 1, 0] if amount = 10`, () => {
+    expect(getCoinCombination(10)).toEqual([0, 0, 1, 0]);
+  });
+
```

Comment:
I would give at list one check of [1] element where are 5 coins.
it(`should return an array: [0, 1, 0, 0] if cents = 5`, () => {
    expect(getCoinCombination(5)).toEqual([0, 1, 0, 0]);
  });